projects/plugins/jetpack/extensions/blocks/dialogue/style.scss

```diff
@@ -1,4 +1,9 @@
+@import '../../shared/styles/gutenberg-base-styles.scss';
+
 .wp-block-jetpack-dialogue {
+	margin-top: 20px;
```

Comment:
Are there any values we can base these on so that it fits with the font sizes of other themes? Something like 1.25em, or if there are any typographic CSS variables that we could use. I can see we're using other pixel values, so it may not be an issue.

Reply:
I had the same question! [Twenty Twenty One has a variable](https://github.com/WordPress/WordPress/blob/967d29d83da3f2d9399d6cf65cf7b62711e92443/wp-content/themes/twentytwentyone/assets/sass/01-settings/global.scss#L91) for `var(--global--spacing-vertical);`, but I don't think that's used by other themes. Global styles [doesn't seem to have a setting](https://github.com/WordPress/gutenberg/blob/966e3d21fb551e9c8253f7c42c88945d16cec541/lib/class-wp-theme-json.php) for vertical spacing (yet). I suppose we could use `rem`, that might be a bit better, though could depend upon the theme. I'm open to better ideas!